packages/@unimodules/react-native-adapter/src/Platform.ts

```diff
@@ -1,7 +1,42 @@
 import { Platform as ReactNativePlatform } from 'react-native';
 
+import { isDOMAvailable } from './environment/browser';
+
+export type PlatformOSType =
+  | 'ios'
+  | 'android'
+  | 'web'
+  | 'native'
+  | 'macos'
+  | 'windows'
+  | 'electron';
+
+export type PlatformSelect = <T>(
+  specifics:
+    | ({ [platform in PlatformOSType]?: T } & { default: T })
+    | { [platform in PlatformOSType]: T }
+) => T;
```

Comment:
I don't think this type is quite right: if you specify android and ios, do you need to specify native?

Reply:
This is the upstream type but with added support for the `native` key.